app/layout.tsx

```diff
@@ -18,11 +18,14 @@ export default function RootLayout({
 }>) {
   const cookiesProvider = cookies();
   const themeCookie = cookiesProvider.get("theme");
-  const theme = themeCookie ? themeCookie.value : "dark";
+  const theme = themeCookie ? themeCookie.value : "light";
```

Comment:
Why are we using cookies in the first place? Doesn't make sense to store such info in cookies, localStorage would make more sense.